### PR TITLE
#1447-fix the wrong mapping and adding resoure for dotnet core, close…

### PR DESCRIPTION
**Pull request by Mosheh**
…s #1447

### CODE REVIEW

File: test/FluentMigrator.Tests/appsettings.json [hunk 1]
```diff
@@ -53,10 +53,10 @@
             "IsEnabled": false,
             "SupportedPlatforms": "x64"
         },
-        "Hana": {
-            "ConnectionString": "Server=Server:Port;Current Schema='\"FluentMigrator\"';UserName=;Password=;",
-            "IsEnabled": false,
-            "SupportedPlatforms": "x64"
-        }
+      "Hana": {
+        "ConnectionString": "Server=Server=192.168.1.200:30015; current schema=CONTRACTONE_WS; UID=SYSTEM; password=Qt7k7dp=y;",
```
Comment (fubar-coder) [inline test/FluentMigrator.Tests/appsettings.json:57]: The test appsettings should (IMHO) not be updated.

**Comment by jzabroski** [comment]
Please delete your PR and rebase it so sensitive info isn't there

**Comment by jzabroski** [comment]
The sensitive info is still in the commit history

**Comment by Mosheh** [comment]
I'm closing this to fullfill a request from Mr. @jzabroski

**Pull request closed without merging**